dbt/README.md

```diff
@@ -494,63 +468,142 @@ We use the following pattern to determine where to define each column descriptio
-We plan to change this once unit testing is released, but for now, make sure that
-any new tests you write are data tests and not unit tests.
+### Data tests
 
-### Adding data tests
+We implement data tests using [dbt tests](https://docs.getdbt.com/docs/build/tests)
+to check that hard-and-fast assumptions about our raw data are correct. We prefer adding tests
+inline in `schema.yml` config files using [generic
+tests](https://docs.getdbt.com/best-practices/writing-custom-generic-tests),
+rather than [singular
+tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests).
 
-There are two types of data tests that we support:
+There are two subtypes of data tests that we support:
 
 1. **QC tests** confirm our assumptions about iasWorld data and are run at
    scheduled intervals to confirm that iasWorld data meets spec
 2. **Non-QC tests** confirm all other assumptions about data sources outside
    of iasWorld, and are run in an ad hoc fashion depending on the needs of
    the transformations that sit on top of the raw data
 
-#### Adding QC tests
+**QC tests** can be run using the [`dbt test`
+command](https://docs.getdbt.com/reference/commands/test) and their output can be
+transformed for analysis using the [`transform_dbt_test_results`
+script](https://github.com/ccao-data/data-architecture/blob/master/dbt/scripts/transform_dbt_test_results.py).
+This script reads the metadata for the most recent `dbt test` run and outputs a number of
+different artifacts with information about the tests:
+
+* An Excel workbook with detailed information on each failure to aid in resolving
+  data problems
+* Parquet files representing metadata tables that can be uploaded to S3 for aggregate
+  analysis
+
+**Non-QC tests** are much less common than QC tests in our test suite. We don't currently
+have a great process for handling them, so there is not much to document yet. If you
+are being asked to add a test that appears to be a non-QC test, meaning that it
+does not confirm our assumptions about iasWorld data, double check with the person who
+assigned the test to you and ask them when and how the test should be run so that its
+attributes can be set accordingly.
+
+#### Running QC tests
+
+There are two instances when QC tests typically run:
+
+1. Once per day by the [`test-dbt-models` GitHub
+   workflow](https://github.com/ccao-data/data-architecture/actions/workflows/test_dbt_models.yaml),
+   which pushes parquet output to S3 in order to support our analysis of test failures over time
+2. On demand by a Data team member whenever Tia, Tom, or another Valuations staff member requests
+   a copy of the Excel workbook for a township, usually right before the town closes
+
+Since the first instance is a scheduled job that requires no intervention, the following
+steps describe how to respond to a request from Tia, Tom, or other Valuations staff for
+a fresh copy of the test failure output before town closing.
```

Comment:
**todo (blocking):** Let's just use "Valuations staff member" instead of names.

Reply:
Makes sense, I switched up both references in 2571113f66dd2df01dffec0599df7d79dff89bcd.